charger/em2go.go

```diff
@@ -158,8 +176,32 @@ func (wb *Em2Go) Enable(enable bool) error {
 	b := make([]byte, 2)
 	binary.BigEndian.PutUint16(b, map[bool]uint16{true: 1, false: 2}[enable])
 
-	_, err := wb.conn.WriteMultipleRegisters(em2GoRegChargeCommand, 1, b)
-	return err
+	if _, err := wb.conn.WriteMultipleRegisters(em2GoRegChargeCommand, 1, b); err != nil {
+		return err
+	}
+
+	// Workaround
+	if wb.workaround {
+		//Get wanted phases set in lp and send to charger
+		var phases int
+		if wb.lp != nil {
```

Comment:
Falls notwendig sollte hier auf die in phases1p3p gesetzten Phases zurück gegriffen werden. Ladepunkt ist nicht notwenig/ hat Seiteneffekte.

Reply:
Soll ich dafür mal einen Vorschlag machen?

Reply:
Sehr gerne, ich weiß nicht wie wir die soll Phasen auslesen können wenn nicht aus dem charger (der sie nicht speichert wenn nicht Enabled und nicht aus dem lp.

Reply:
evcc würde die Phasen 1x setzen wenn es sich unsicher ist und geht bis dahin von "unklar" aus. Deshalb sollte es reichen sich den gesetzten Wert im Charger zu merken.

Reply:
Siehe https://github.com/evcc-io/evcc/pull/16201/commits/72daf4978a74f3da6a78e3a4159e11e138d9c9d6